oaknational/Oak-Web-Application
# style: fetch correct image sizes on blog listing

## Description

- sets sizes attr on blog list item images
  - annoyingly this involves duplicating where we specify the image size, but can't see a better way whilst also achieving performance gain of small image
- adds deviceSizes to next.config (https://github.com/vercel/next.js/issues/18413#issuecomment-775591999) 
- sets background color on CMS image for loading placeholder pastelTurqoise [sic]

## Issue(s)

Progresses #583 

## How to test

1. Go to {cloud link}
2. Should see light blue placeholder before images load
3. Blog image should load a small image (256px width on desktop, 100vw on mobile breakpoint)

## Checklist
